src/sql/azdata.d.ts

```diff
@@ -2579,15 +2579,15 @@ declare module 'azdata' {
 		 * Prompts the user to enter account information.
 		 * Returns an error if the user canceled the operation.
 		 */
-		prompt(): Thenable<Account | PromptFailedResult>;
+		prompt(): Thenable<Account | ProviderError>;
```

Comment:
Shouldn't all these stay as PromptFailedResult? That will include the new ProviderError properties you added - and these will all have the canceled property on them so ProviderError isn't correct.